app/src/main/graphql/com/example/rocketreserver/schema.graphqls

```diff
@@ -0,0 +1,103 @@
+type Query {
```

Comment:
Woops this missing!  But then we should delete `schema.json` then.

Reply:
Right! I just removed it